base/db/R/get.machine.info.R

```diff
@@ -0,0 +1,83 @@
+#' Get machine information from db
+#' @param host host information
+#' @param input.args input args for existing records
+#' @param input.id input id for existing records
+#' @param con database connection
+#'
+#' @return list of machine, input, and dbfile records
+#' @author Betsy Cowdery, Michael Dietze, Ankur Desai, Tony Gardella, Luke Dramko
+
+get.machine.info <- function(host, input.args, input.id = NULL, con = NULL) {
+
+  machine.host.info <- get.machine.host(host, con = con)
+  machine.host <- machine.host.info$machine.host
+  machine <- machine.host.info$machine
+
+  if (nrow(machine) == 0) {
+    PEcAn.logger::logger.error("machine not found", host$name)
+    return(NULL)
+  }
+
+  if (missing(input.id) || is.na(input.id) || is.null(input.id)) {
```

Comment:
Don't need the `missing()` here because get.machine.info now defaults it to NULL
```suggestion
  if (is.na(input.id) || is.null(input.id)) {
```